include/sound/sof/ext_manifest.h

```diff
@@ -61,6 +61,7 @@ enum sof_ext_man_elem_type {
 	SOF_EXT_MAN_ELEM_WINDOW			= SOF_IPC_EXT_WINDOW,
```

Comment:
The language is a bit complex in the commit message.
"like features enabled or disabled state or buffers size" -> "like feature availability, buffer size limits and similar properties" ?

"Extracted get_config_value() will be reused during platform specific configuration dictionary parsing in future."
This leaves more questions than it answers, and I wonder if it's better to just leave this sentence out.

Reply:
First part is changed, second one is completely deleted together with `get_config_value()` implementation.